### PR TITLE
- Reduce visibility of ClusterState and Distribution.

### PR DESCRIPTION
**Pull request by baldersheim**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by baldersheim** [comment]
No semantic changes

File: storage/src/vespa/storage/distributor/distributorcomponent.h [hunk 1]
```diff
@@ -11,8 +11,11 @@
 #include <vespa/storage/storageutil/utils.h>
 #include <vespa/storageapi/messageapi/storagecommand.h>
 #include <vespa/storageapi/buckets/bucketinfo.h>
-#include <vespa/vdslib/state/clusterstate.h>
 
+namespace staorage::lib {
```
Comment (vekterli) [inline storage/src/vespa/storage/distributor/distributorcomponent.h:15]: typo: `staorage` -> `storage`

**Review by vekterli — APPROVED** [review]
:+1:

File: storage/src/vespa/storage/distributor/distributorcomponent.h (hunk 1 above)
Comment (baldersheim) [reply storage/src/vespa/storage/distributor/distributorcomponent.h]: Fixed

**Pull request merged**